src/server/api/routers/expenses.ts

```diff
@@ -0,0 +1,185 @@
+import { z } from "zod";
+import { createTRPCRouter, protectedProcedure } from "@/server/api/trpc";
+import { TRPCError } from "@trpc/server";
+
```

Comment:
Nice! creating the schema outside is really great. It's good too because you can infer a type from the schema for later use.